### PR TITLE
enhance(macros/CSSInfo): add "Animation type" link

### PR DESCRIPTION
**Pull request by mostafaroshdy1**
## Summary

This PR added the enchantment in this issue #8721 
### Problem

The ''Animation type" string was not a hyperlink, so i made it an anchor tag referring to the related page

### Solution

- I have created a file in macros named AnimationType and added the localString as i found them in the official page [here](https://developer.mozilla.org/en-US/docs/Web/CSS/height#formal_definition) (in every other language)
- 

---

### Before

image

### After
image

---

## How did you test this change?
Locally using ```yarn dev```

### CODE REVIEW

File: kumascript/macros/CSSInfo.ejs [hunk 1]
```diff
@@ -283,7 +283,7 @@ if (name === "preview-wiki-content") {
     if (!atRule) {
         properties = properties.concat({
             name: "animationType",
-            label: localize(localStrings, "animationType")
+            label: await template("animationType")
```
Comment (caugner) [inline kumascript/macros/CSSInfo.ejs:286]: Let's avoid introducing a new macro here. Can you add the local string to this macro and build the link in line here?

**Review by caugner — CHANGES_REQUESTED** [review]

File: kumascript/macros/CSSInfo.ejs (hunk 1 above)
Comment (mostafaroshdy1) [reply kumascript/macros/CSSInfo.ejs]: done , you can check the latest commit , thanks

File: kumascript/macros/CSSInfo.ejs [hunk 2]
```diff
@@ -283,7 +283,7 @@ if (name === "preview-wiki-content") {
     if (!atRule) {
         properties = properties.concat({
             name: "animationType",
-            label: localize(localStrings, "animationType")
+            label: `<a href="/${env.locale}/docs/Web/CSS/CSS_animated_properties">${localize(localStrings, "animationType")}</a>`
```
Comment (caugner) [inline kumascript/macros/CSSInfo.ejs:286]: Thank you, this makes me realize we should probably use `smartLink()` here. Can you try locally if this works?

```suggestion
            label: web.smartLink(`/${env.locale}/docs/Web/CSS/CSS_animated_properties`, null, localize(localStrings, "animationType"))
```

**Review by caugner — CHANGES_REQUESTED** [review]

File: kumascript/macros/CSSInfo.ejs (hunk 2 above)
Comment (mostafaroshdy1) [reply kumascript/macros/CSSInfo.ejs]: it worked just fine , thanks for the suggestion

**Review by caugner — APPROVED** [review]

**Comment by mostafaroshdy1** [comment]
@caugner 

I have updated the commits signature

File: yarn.lock [hunk 3]
```diff
@@ -11360,11 +11360,12 @@ open@^8.4.0:
     define-lazy-prop "^2.0.0"
     is-docker "^2.1.1"
     is-wsl "^2.2.0"
-
+
```
Comment (caugner) [inline yarn.lock:11363]:
```suggestion

```

File: yarn.lock [hunk 4]
```diff
@@ -11360,11 +11360,12 @@ open@^8.4.0:
     define-lazy-prop "^2.0.0"
     is-docker "^2.1.1"
     is-wsl "^2.2.0"
-
+    
 openai@^4.38.1:
   version "4.38.1"
   resolved "https://registry.yarnpkg.com/openai/-/openai-4.38.1.tgz#375ba1e2ff9fcd6434114bfeffa7a0ef35d74b78"
   integrity sha512-nmSKE9O2piuoh9+AgDqwGHojIFSxToQ2jJqwaxjbzz2ebdD5LYY9s+bMe25b18t4QEgvtgW70JfK8BU3xf5dRw==
+
```
Comment (caugner) [inline yarn.lock:11368]:
```suggestion
```

**Review by caugner — CHANGES_REQUESTED** [review]

File: yarn.lock [hunk 5]
```diff
@@ -11360,7 +11360,6 @@ open@^8.4.0:
     define-lazy-prop "^2.0.0"
     is-docker "^2.1.1"
     is-wsl "^2.2.0"
```
Comment (caugner) [inline yarn.lock:11362]:
```suggestion
    is-wsl "^2.2.0"

```

**Review by caugner — APPROVED** [review]

**Pull request merged**